src/webportal/deploy/webportal.yaml.template

```diff
@@ -33,31 +33,31 @@ spec:
       hostPID: false
       containers:
       - name: webportal
-        image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}webportal:{{ clusterinfo['dockerregistryinfo']['docker_tag'] }}
+        image: {{ cluster_cfg['cluster']['docker-registry']['prefix'] }}webportal:{{ cluster_cfg['cluster']['docker-registry']['tag'] }}
         imagePullPolicy: Always
         env:
         - name: REST_SERVER_URI
-          value: {{ clusterinfo['webportalinfo']['rest_server_uri'] }}
+          value: {{ cluster_cfg['rest-server']['uri'] }}
         - name: PROMETHEUS_URI
-          value: {{ clusterinfo['webportalinfo']['prometheus_uri'] }}
+          value: {{ cluster_cfg['prometheus']['url'] }}
         - name: YARN_WEB_PORTAL_URI
-          value: {{ clusterinfo['webportalinfo']['yarn_web_portal_uri'] }}
+          value: http://{{ cluster_cfg['hadoop-resource-manager']['master-ip']] }}:8088 
         - name: GRAFANA_URI
-          value: {{ clusterinfo['webportalinfo']['grafana_uri'] }}
+          value: {{ cluster_cfg['grafana']['url'] }}
         - name: K8S_DASHBOARD_URI
-          value: {{ clusterinfo['webportalinfo']['k8s_dashboard_uri'] }}
+          value: http://{{ cluster_cfg['kubernetes']['dashboard-host'] }}:8080
         - name: K8S_API_SERVER_URI
-          value: {{ clusterinfo['webportalinfo']['k8s_api_server_uri'] }}
+          value: http://{{ cluster_cfg['kubernetes']['api-servers-ip'] }}:9090
```

Comment:
is this 9090? default 8080 is for api, 9090 is for dashboard. Also please change it to [`api-servers-url`](https://github.com/Microsoft/pai/pull/1792/files#diff-edc53779b489ce60c7dc276f0159f4f4R83)